src/core.ttl

```diff
@@ -35,11 +35,11 @@ A Data Space is highly customizable to allow its members to create a space of sh
 okp4core:Dataset a owl:Class ;
   rdfs:label "Dataset"@en ;
   rdfs:comment """
-  `Dataset` denotes the collection of related sets of information, encoded in a defined structure, as published by the data provider.
-  This includes a wide variety of formats, for example: csv files, images, videos, databases, etc.
+  `Dataset` represents the group of related data that is organized and presented in a specific format by the provider. This data can take many forms, 
+  including csv files, images, videos, databases, and more, as well as data sources such as databases and APIs.
```

Comment:
> including csv files, images, videos, `databases`, and more, as well as data sources such as `databases` and APIs.  

Databases is mentioned 2 times, is it done on purpose ?

Reply:
Correct. I'll fix that. :+1: